macros/GroupData.json

```diff
@@ -96,6 +97,14 @@
                             "Element: cut",
                             "Element: paste" ]
         },
+        "Content Index API": {
+            "overview":   [ "Content Index API" ],
+            "interfaces": [ "ContentIndex", "ContentIndexEvent" ],
```

Comment:
Put each interface on a single line, and line up the opening quote marks of each one.